src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -82,6 +86,11 @@ public String toString() {
         public boolean isFirstGroupedByGiver() {
             return this == GIVER_QUESTION_RECIPIENT;
         }
+        
+        public boolean isGroupedBySecondaryParticipant() {
+            return this == RECIPIENT_GIVER_QUESTION;
+        }
```

Comment:
Is there a better name for this?
Maybe something more specific signifying what it looks in the frontend?